WireDomain/Sources/WireDomain/Providers/InitialSyncBuilderProtocol.swift

```diff
@@ -26,6 +26,14 @@ public protocol InitialSyncBuilderProtocol {
     // due to `any InitialSyncProtocol?`
     typealias Sync = any InitialSyncProtocol
 
-    func build() throws -> Sync
+    func buildInitialSync() throws -> Sync
+
+}
+
+extension ClientSessionComponent: InitialSyncBuilderProtocol {
+
+    public func buildInitialSync() throws -> any InitialSyncProtocol {
+        initialSync
+    }
```

Comment:
suggestion: move to own file ClientSessionComponent+InitialSyncBuilderProtocol.swift

Reply:
done 16617699e92bb879265fbee8d82f6f71c2421e2d